api/comments/comments.spec.js

```diff
@@ -0,0 +1,127 @@
+const req = require('supertest');
+const server = require('../../server');
+const db = require("../../data/dbConfig");
+
+beforeEach(() => {
+  return db("comments").truncate();
+});
+
+describe("The Comments Router", () => {
+  it('should start testing environment', () => {
+    const res = process.env.DB_ENV;
+    expect(res).toBe('testing');
+  });
+
+  describe("GET /", () => {
+    it("should return an array of comments", async () => {
+      const comments = [{
```

Comment:
If you pull this out into the top scope and call it in the tests it can reduce lines and make things more readable.